### ISSUE TITLE
Daily templates with :olp option and "Heading not found on level 1"

### ISSUE THREAD

**Issue by czikus** [org-roam/org-roam#1275]
### Description

I'm trying to follow the manual and configure a template that would add a daily entry into a specific heading of the daily file for a specific day.

My template looks as follows:
```
("d" "default" entry #'org-roam-capture--get-point
           "* %?"
           :file-name "Daily/%<%Y-%m-%d>"
           :head "#+TITLE: %<%Y-%m-%d>\n"
           :olp ("Daily")
           )
```

However, this results in `org-roam-capture--get-point: Heading not found on level 1: Daily`.

#### Steps to Reproduce

1. Load Emacs
2. Set `org-roam-dailies-capture-templates` to the template above.
3. Run `org-roam-dailies-capture-today`

#### Expected Results

New file created for the day containing a heading called "Daily" with the new entry in it.

#### Actual Results

 `org-roam-capture--get-point: Heading not found on level 1: Daily`.

### Environment

- Emacs: GNU Emacs 27.1 (build 3, x86_64-pc-linux-gnu, GTK+ Version 3.22.30, cairo version 1.15.10)
 of 2020-09-08
- Framework: Doom
- Org: Org mode version 9.5 (nil @ /home/user/.emacs.d/.local/straight/build-27.1/org-mode/)
- Org-roam: 1.2.2

**Comment by zaeph** [comment]
When I merged #978, it was necessary to include the heading inside the template to avoid this error message.  In #1270, @jethrokuan merged a patch that removed that need.

Could you update org-roam and try again?

**Comment by czikus** [comment]
Works like a charm! Thanks!!
